arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java

```diff
@@ -361,4 +361,28 @@ private void mergeResult(String operation, String category, ReplayCompareResult
     diffResult.setReplayTime(replayTime);
     diffResult.setInstanceId(instanceId);
   }
+
+  private boolean checkEntryPointCallMissing(List<CategoryComparisonHolder> comparisonHolderList){
+    if (CollectionUtils.isEmpty(comparisonHolderList)) {
+      return false;
+    }
+    // check if all entry point call missing or new call
+    for (CategoryComparisonHolder comparisonHolder : comparisonHolderList) {
+      if (CollectionUtils.isNotEmpty(comparisonHolder.getRecord()) &&
+              CollectionUtils.isEmpty(comparisonHolder.getReplayResult()) &&
+              comparisonHolder.getRecord().stream().allMatch(CompareItem::isEntryPointCategory)) {
+        LOGGER.warn("[[title=checkEntryPointCallMissing]]entry point call missing, case compareKey" +
+                " :{}", comparisonHolder.getRecord().get(0).getCompareKey());
+        return true;
+      }
+      if (CollectionUtils.isNotEmpty(comparisonHolder.getReplayResult()) &&
+              CollectionUtils.isEmpty(comparisonHolder.getRecord()) &&
+              comparisonHolder.getReplayResult().stream().allMatch(CompareItem::isEntryPointCategory)) {
+        LOGGER.warn("[[title=checkEntryPointCallMissing]]entry point new call, case compareKey" +
+                " :{}", comparisonHolder.getReplayResult().get(0).getCompareKey());
+        return true;
+      }
+    }
+    return false;
+  }
```

Comment:
Recommended to put these logic into `PrepareCompareSourceRemoteLoader#buildWaitCompareList`